soh/src/overlays/actors/ovl_Boss_Dodongo/z_boss_dodongo.c

```diff
@@ -69,7 +69,7 @@ static u8 sMaskTex16x32[16 * 32] = { { 0 } };
 static u8 sMaskTex32x16[32 * 16] = { { 0 } };
 static u8 sMaskTex8x8[8 * 8] = { { 0 } };
 static u8 sMaskTex8x32[8 * 32] = { { 0 } };
-static u8 sMaskTexLava[32 * 64] = { { 0 } };
+static u8 sMaskTexLava[LAVA_TEX_HEIGHT * LAVA_TEX_HEIGHT] = { { 0 } };
```

Comment:
Is this supposed to be height * height instead of width * height?

Reply:
oops